UI/DevScreens.cpp

```diff
@@ -845,6 +845,7 @@ void SystemInfoScreen::CreateTabs() {
 	LinearLayout *internals = AddTab("DevSystemInfoInternals", si->T("Internals"));
 	CreateInternalsTab(internals);
 #endif
+	delete tabHolder;
```

Comment:
Like the one below, this is a different kind of bug. I'll fix it separately, you can delete this line.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18967/commits/9c3c23daa6d3a70921f3aeb3560c81b497065eb4